### PR TITLE
new plugin

### PR DESCRIPTION
**Pull request by naplaya**

### CODE REVIEW

File: items/public-announcements [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Public Announcements",
+	"version": "1.0",
+	"author_username": "aniika",
+	"release_date": "13.04.2019",
```
Comment (dignajar) [inline items/public-announcements]: Please change the date format to `yyyy-mm-dd`

File: items/public-announcements [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	"license": "MIT",
+	"price_usd": "0",
+	"download_url": "https://github.com/naplaya/public-announcements/archive/master.zip",
```
Comment (dignajar) [inline items/public-announcements]: Please create a release in Github with the version and update the links for download.

File: items/public-announcements [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	"demo_url": "bc-schoellbronn.de",
+	"information_url": "https://github.com/naplaya/public-announcements",
+	"description": "This plugins allows to display some announcements for example a server downtime on a specific day",
+	"features": {
+		"0": "Customize colors of the announcements",
+		"1": "Set start and end date of the announcement",
+		"2": "Fully customizable style"
+	},
```
Comment (dignajar) [inline items/public-announcements]: remove the comma, need to be a valid json file

File: items/public-announcements (hunk 2 above)
Comment (naplaya) [reply items/public-announcements]: sry dont understand what you mean with update link. this link download the file  if visited in browser

Comment (dignajar) [reply items/public-announcements]: The user is going to download the master branch from the repository with that link, so if you do push to the master branch is NOT going to be consistent with the version 1.0. You can create a release like I do in Bludit https://github.com/bludit/bludit/releases and you are going to have a link only for the version 1.0. You get my point ?

**Pull request merged**